caluma/caluma_analytics/serializers.py

```diff
@@ -67,24 +79,6 @@ def _validate_field_exists(self, data):
                 }
             )
 
-    def _validate_uniqueness(self, data):
-        existing_fields = data["table"].fields.all()
-        if self.instance:
-            existing_fields = existing_fields.exclude(pk=self.instance.pk)
-
-        if existing_fields.filter(alias=data["alias"]).exists():
-            raise exceptions.ValidationError(
-                {"alias": ["Cannot use the same alias twice within the same table"]}
-            )
-        if existing_fields.filter(data_source=data["data_source"]).exists():
-            raise exceptions.ValidationError(
-                {
-                    "dataSource": [
-                        "Cannot use the same data source twice within the same table"
-                    ]
-                }
-            )
-
```

Comment:
Why isn't this necessary anymore?

Reply:
In other parts of Caluma, we let the ORM do such uniqueness checks, which has the same result, just with a bit less-nice of an error message. But since end-users don't see this either way, it's not needed

Reply:
:+1: